composer.lock

```diff
@@ -435,7 +435,7 @@
                 "cache",
                 "caching"
             ],
-            "time": "2017-01-20 09:27:09"
+            "time": "2017-01-20T09:27:09+00:00"
```

Comment:
https://stackoverflow.com/questions/41552755/composer-lock-timestamp-format says it's the `1.3.0` version, probably we have been using an old version on recommendations so far.